package/MDAnalysis/coordinates/PDB.py

```diff
@@ -651,7 +699,30 @@ def _write_pdb_header(self):
             self.REMARK(*remarks)
         except AttributeError:
             pass
-        self.CRYST1(self.convert_dimensions_to_unitcell(u.trajectory.ts))
+
+        # FIXME: Values for meaningless cell dimensions are not consistent.
+        # FIXME: See Issue #2698. Here we check for both None and zeros
+        if u.dimensions is None or np.allclose(u.dimensions, np.zeros(6)):
+            # Unitary unit cell by default. See PDB standard:
+            # http://www.wwpdb.org/documentation/file-format-content/format33/sect8.html#CRYST1
+            self.CRYST1(np.array([1.0, 1.0, 1.0, 90.0, 90.0, 90.0]))
+
+            # Add CRYST1 REMARK (285)
+            # The SCALE record is not included
```

Comment:
Thought here, do we/should we be accounting for the SCALE record in the PDBReader?

Reply:
Looks like we should add the `SCALE` record as well?

> For NMR, fiber diffraction, and EM entries, SCALE is given as an identity matrix with no translation.

Reply:
With further thought, this wouldn't be a bad thing to do here, As per your comment https://github.com/MDAnalysis/mdanalysis/pull/2685#issuecomment-635984874 it would be good to raise an issue about this for reading / non-default writing if possible.